### DISCUSSION TITLE
Add a keyboard shortcut to change planets

### DISCUSSION THREAD

**Discussion by imariasantos** [Stellarium/stellarium#1741] [Q&A] [answered]
To change the planet from which I am viewing the sky, I can go into the "Location" window and select a different planet, say Mars. But I don't know how to do this from the keyboard. 

I would like to add a keyboard shortcut to switch my location to Mars. How can I do that?

Cheers

**Comment by alex-w** [comment, marked as answer]
Select the Mars and press Ctrl+G (described in Help window (shortcut: F1) and in the Stallarium User Guide).

Other way: write a script, which contains the code to switch a location and assign shortcut to the script.

**Reply by imariasantos** [reply]
Thank you for your answer. What would such a script look like?

**Reply by alex-w** [reply]
Stellarium User Guide + examples: https://github.com/Stellarium/stellarium/tree/master/scripts + user's scripts: https://stellarium.org/scripts.html + scripting documentation: https://stellarium.org/doc/0.21/scripting.html

**Comment by axd1967** [comment]
@imariasantos did you try following:
1. select a target planet
2. press CTRL-G
